bindings/nodejs/lib/client/client.ts

```diff
@@ -238,7 +266,11 @@ export class Client {
     }
 
     /**
-     * Prepare a transaction for signing
+     * Prepare a transaction for signing.
```

Comment:
```suggestion
     * Prepare a transaction.
```
Any opinion on this? With `for signing` to me it sounds like we have a transaction and do something to it so we can sign it, but this function creates the transaction and signing is just the next step that needs to be done

Reply:
I think it's fine but no strong feelings.